UPGRADING.md

```diff
@@ -9,6 +9,60 @@ The `rack_response` method has been deprecated and the `error_response` method h
 
 See [#2414](https://github.com/ruby-grape/grape/pull/2414) for more information.
 
+#### Change in parameters precedence
+
+When using together with `Grape::Extensions::Hash::ParamBuilder`, `route_param` takes higher precedence
+over a regular parameter defined with same name, which now matches the default param bulider behavior.
```

Comment:
typo: builder

Reply:
Also remove the wrapping in markdown, https://code.dblock.org/2021/06/07/to-wrap-or-not-to-wrap-in-markdown.html

Reply:
Thanks for spotting the typo!